ios/ReactNativeCameraKit/CameraView.swift

```diff
@@ -262,15 +262,21 @@ class CameraView: UIView {
         }
     }
 
-    private func writeCaptured(imageData: Data, 
+    private func writeCaptured(imageData: Data,
+                               thumbnailData: Data?,
```

Comment:
Yeah, it's just laying some preparation for later changes. I had it working locally but have not committed enabling it since it needs several other features enabled and must be an opt-in feature. We can leave it in for now.